zucchini-ui-frontend/src/failures/components/FailuresTableContainer.js

```diff
@@ -0,0 +1,21 @@
+import {connect} from 'react-redux';
+import {createSelector, createStructuredSelector} from 'reselect';
+
+import FailuresTable from './FailuresTable';
+
+const selectFailures = createSelector(
```

Comment:
Selector too vague. You should return `failules.failures` from the Redux store. Your selector should look like :

```
const selectFailures = createSelector(
  state => state.failures.failures,
  failures => failures,
);
```